scripts/ci/perf_test.sh

```diff
@@ -1,45 +1,26 @@
 set -ex -o pipefail
 
-case "${TEST_FORMAT}" in
-  http|flux-http)
-    ;;
-  *)
-    >&2 echo "Unknown query format: ${TEST_FORMAT}"
-    exit 1
-    ;;
-esac
+source vars.sh
 
-# get latest ubuntu 20.04 ami for us-west-2
-ami_id=$(aws --region us-west-2 ssm get-parameters --names /aws/service/canonical/ubuntu/server/20.04/stable/current/amd64/hvm/ebs-gp2/ami-id --query 'Parameters[0].[Value]' --output text)
+# get latest ubuntu 21.10 ami for us-west-2
+ami_id=$(aws --region us-west-2 ssm get-parameters --names /aws/service/canonical/ubuntu/server/21.10/stable/current/amd64/hvm/ebs-gp2/ami-id --query 'Parameters[0].[Value]' --output text)
```

Comment:
Bump the version here instead of trying to slam in `libc6` during `run_perftest.sh`.